pkg/kubectl/cmd/util/helpers.go

```diff
@@ -408,16 +408,6 @@ func AddValidateOptionFlags(cmd *cobra.Command, options *ValidateOptions) {
 
 func AddOpenAPIFlags(cmd *cobra.Command) {
 	cmd.Flags().Bool("openapi-validation", false, "If true, use openapi rather than swagger for validation")
-	cmd.Flags().String("schema-cache-dir",
```

Comment:
1.7 shipped with the "schema-cache-dir" flag... cannot just drop it in favor of "cachedir"